### PR TITLE
More permissive input filters to not block entity embeds

### PR DESCRIPTION
**Pull request by AronNovak**
So we won't face with
https://www.drupal.org/project/entity_embed/issues/2596487#comment-10521562
next time when we'd like to use entity embed at a new client project.

### CODE REVIEW

**Comment by AronNovak** [comment]
Let me know if it worth a test.

File: config/sync/filter.format.full_html.yml [hunk 1]
```diff
@@ -40,6 +40,6 @@ filters:
...
...ir dirname disabled download draggable enctype for form formaction headers height hidden high href hreflang http-equiv id ismap kind label lang list loop low max maxlength media method min multiple muted name novalidate open optimum pattern placeholder poster preload readonly rel required reversed rows rowspan sandbox scope selected shape size sizes span spellcheck src srcdoc srclang srcset start step style tabindex target title translate type usemap value width wrap> <wbr accept accept-charset accesskey action align alt async autocomplete autofocus autoplay bgcolor border charset checked cite class color cols colspan content contenteditable controls coords data data-* datetime default defer dir dirname disabled download draggable enctype for form formaction headers height hidden high href hreflang http-equiv id ismap kind label lang list loop low max maxlength media method min multiple muted name novalidate open optimum pattern placeholder poster preload readonly rel required reversed rows rowspan sandbox scope selected shape size sizes span spellcheck src srcdoc srclang srcset start step style tabindex target title translate type usemap value width wrap> <drupal-entity * data-*>'
```
Comment (amitaibu) [inline config/sync/filter.format.full_html.yml]: Is it correct having a second set of tags here `<drupal-entity * data-*>` - and not having it inside the first tag?

Comment (AronNovak) [reply config/sync/filter.format.full_html.yml]:
```
 aaron   entity-embed-filter  ~  gizra  drupal-starter  cat config/sync/filter.format.full_html.yml  | grep -o drupal-entity | wc -l
1
```

I double-checked, `<drupal-entity>` is not a duplicate in the list. Is it what you referred to?

Comment (amitaibu) [reply config/sync/filter.format.full_html.yml]: I mean, it's on a separate tag.

image

**Comment by amitaibu** [comment]
Thanks

**Pull request merged**